cmd/app.go

```diff
@@ -145,6 +145,10 @@ func AppCommand() cli.Command {
 						Name:  "version",
 						Usage: "Version of the template to use",
 					},
+					cli.BoolFlag{
+						Name:  "no-prompt",
+						Usage: "Suppress asking questions and use the default values when required answers are not provided",
```

Comment:
what is the default value?

Reply:
False for bool value

Reply:
shall we list the default value in help? there is a cli flag for it, called default value or something